platform/linux-generic/arch/x86/odp_time_cpu.c

```diff
@@ -15,24 +14,67 @@
 
 #include <time.h>
 
-/* Measure TSC frequency. Frequency information registers are defined for x86,
- * but those are often not enumerated. */
+static int nsleep(uint64_t nsec)
```

Comment:
Maybe this could be `nwait` or something similar as no actual sleeping is done in the function.

Reply:
Good point. Renamed to nwait in v2 and also renamed sleep_nsec to wait_nsec.